### DISCUSSION TITLE
Question about default_geometry

### DISCUSSION THREAD

**Discussion by abhaysrinivas3012** [pybamm-team/PyBaMM#2442] [Q&A] [answered]
Hi,

I am new to Pybamm. I am having a hard time trying to figure out how the geometry of the default cell is defined. The output of the model.default_geometry is 
{'negative electrode': {'x_n': {'min': 0,
   'max': Division(-0x65de7ddab81ccac5, /, children=['Negative electrode thickness [m]', 'Negative electrode thickness [m] + Separator thickness [m] + Positive electrode thickness [m]'], domains={})}},
 'separator': {'x_s': {'min': Division(-0x65de7ddab81ccac5, /, children=['Negative electrode thickness [m]', 'Negative electrode thickness [m] + Separator thickness [m] + Positive electrode thickness [m]'], domains={}),
   'max': Addition(0x59adfffc65fff607, +, children=['Negative electrode thickness [m] / (Negative electrode thickness [m] + Separator thickness [m] + Positive electrode thickness [m])', 'Separator thickness [m] / (Negative electrode thickness [m] + Separator thickness [m] + Positive electrode thickness [m])'], domains={})}},
 'positive electrode': {'x_p': {'min': Addition(0x59adfffc65fff607, +, children=['Negative electrode thickness [m] / (Negative electrode thickness [m] + Separator thickness [m] + Positive electrode thickness [m])', 'Separator thickness [m] / (Negative electrode thickness [m] + Separator thickness [m] + Positive electrode thickness [m])'], domains={}),
   'max': 1}},
 'negative particle': {'r_n': {'min': 0, 'max': 1}},
 'positive particle': {'r_p': {'min': 0, 'max': 1}},
 'current collector': {'z': {'position': 1}}}

Could someone help me understand this

**Comment by rtimms** [comment, marked as answer]
It's probably easier to look at the [source code](https://github.com/pybamm-team/PyBaMM/blob/develop/pybamm/geometry/battery_geometry.py) to understand this. The negative electrode, separator and positive electrode are of thickness L_n, L_s, L_p, respectively. And particles are of radius R_n, R_p. Take a look at Figure 6 of [this paper](https://iopscience.iop.org/article/10.1088/2516-1083/ac7d31?gclid=CjwKCAjw3K2XBhAzEiwAmmgrAhIelwpWeWtXZoW-vkPQbLoAGtxsqS1AUHas3dCY6cgyY64ZKUXC9xoC1ZUQAvD_BwE) for a nice schematic. 

The output above looks a bit messy because it is in dimensionless form (lots of dividing by L_n + L_s + L_p to normalize the thickness).
